src/lib/helpers/ArrowHelper.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+  import type { ArrowHelperProperties, Object3DProperties } from '$lib/types/components';
+  import { ArrowHelper as ThreeArrowHelper } from 'three'
+  import Object3DInstance from '../instances/Object3DInstance.svelte'
+
+  // Object3DInstance
+  export let position: Object3DProperties['position'] = undefined
```

Comment:
These props can also be typed from `ArrowHelperProperties` as these types are *extending* the `Object3DInstanceProperties` types.